client/src/modules/list/components/Items/ItemsList.jsx

```diff
@@ -47,32 +80,11 @@ class ItemsList extends PureComponent {
     }
   };
 
-  handleLimitUpdate = () => {
-    const {
-      archived: archivedItems,
-      done: doneItems,
-      match: {
-        params: { id: listId }
-      },
-      updateLimit
-    } = this.props;
-    const { limit: currentLimit } = this.state;
-    const limit = {};
-
-    if (archivedItems) {
-      limit.archivedLimit = currentLimit;
-    } else if (doneItems) {
-      limit.doneLimit = currentLimit;
-    } else {
-      limit.unhandledLimit = currentLimit;
-    }
-
-    updateLimit(limit, listId);
-  };
+  // TODO: Remove updateItem action
```

Comment:
restore handleLimitUpdate is necessary to decide if an item is to be created or not (look at `shouldAnimate` helper.